x/tx/CHANGELOG.md

```diff
@@ -33,6 +33,10 @@ Since v0.13.0, x/tx follows Cosmos SDK semver: https://github.com/cosmos/cosmos-
 
 ## [Unreleased]
 
+### API Breaking
+
+* [#20424](https://github.com/cosmos/cosmos-sdk/pull/20424) `decode.Options` now requires a proto codec capable of unmarshalling and resolving types.
```

Comment:
Is it used in v0.50? I don't think it is, we if so we should add an example and make sure v0.50 is updated. Otherwise we'll need a compatibility matrix for x/tx and we don't want that.

Reply:
x/tx/decode is used in v0.50, or which did you mean?

Reply:
Then I missed it (on my phone). What does it mean for people in v0.50 then? Do we need a patch release that supports that behavior change then?

Reply:
sorry, you are correct, it is unused in v0.50.